addons/account_peppol/tools/demo_utils.py

```diff
@@ -78,12 +77,21 @@ def _mock_button_verify_partner_endpoint(func, self, *args, **kwargs):
 
 def _mock_user_creation(func, self, *args, **kwargs):
     func(self, *args, **kwargs)
-    self.write({
-        'account_peppol_proxy_state': 'active',
-    })
+    self.account_peppol_proxy_state = 'sender' if not self.smp_registration else 'smp_registration'
```

Comment:
Cosmetic comment: it's easier to read without negate
```suggestion
    self.account_peppol_proxy_state = 'smp_registration' if self.smp_registration else 'sender'
```